krun.py

```diff
@@ -192,11 +177,10 @@ def main(parser):
         util.print_session_info(config)
         return
 
-    if args.strip_results:
-        util.strip_results(config, args.strip_results)
-        return
+    on_first_invokation = not (os.path.isfile(ManifestManager.PATH) and
```

Comment:
Did we somwhere check that if a manifest file exists, so should a results file, and the other way around?